extended/src/main/java/io/kubernetes/client/extended/generic/options/CreateOptions.java

```diff
@@ -0,0 +1,7 @@
+package io.kubernetes.client.extended.generic.options;
+
+public class CreateOptions {
+  private Boolean dryRun;
+  private Object
+      fieldManager; // TODO: structured field-manager for better server-side apply integeration
```

Comment:
why is this on a newline?

Reply:
it's the TODO, i moved it to another line